client/templates/orderDetails/status/orderReturned/orderReturned.js

```diff
@@ -54,6 +57,13 @@ Template.orderReturned.helpers({
     let orderId = this._id;
     let itemStatus = Session.get('orderReturned-' + orderId)[itemId];
     return itemStatus === 'shipped';
+  },
+  itemType: function (item) {
+    let itemType = item.functionalType;
+    if (itemType === 'variant') {
+      return 'purchased';
```

Comment:
Seen this a couple times now, we should probably chat about what the default should be. My gut says default is purchasable and declared is rental.